pages/common/catimg.md

```diff
@@ -0,0 +1,21 @@
+# catimg
+
+> Image printing in the terminal.
+> See also: `pixterm` and `chafa`.
+> More information: <https://github.com/posva/catimg>.
+
+- Print a JPEG, PNG, or GIF to the terminal:
+
+`catimg {{path/to/file}}`
+
+- Use doubled [r]esolution:
```

Comment:
```suggestion
- Increase the resolution the image is printed by a specified multiplier:
```

Reply:
Yeah, again, it's only possible to set the `-r` flag to 2. It's basically like `-r 2` is the whole flag.

Reply:
```suggestion
- Double the resolution of an image: 
```

Reply:
Thank you!